client/components/app/app.js

```diff
@@ -4,65 +4,111 @@ import VideoPlayer from '../videoPlayer/videoPlayer';
     },
   });
 
   const [currentlyDisplaying, setCurrentDisplay] = useState('VIDEOS');
 
+  const [featuredMedia, setFeaturedMedia] = useState(dummyData.videos[0]);
+
   useEffect(() => {
     const queryString = window.location.search;
     const params = new URLSearchParams(queryString);
     const id = params.get('id');
-    $.ajax(`/games/${id}`, {
-      method: 'GET',
-      success: (results) => {
-        setDummyData(results);
-      },
-      error: (err) => console.error(err),
-    });
-  }, [dummyData]);
+    if (id) {
+      $.ajax(`/games/${id}`, {
+        method: 'GET',
+        success: (results) => {
+          setDummyData(results);
+          setFeaturedMedia(results.videos[0]);
+        },
+        error: (err) => console.error(err),
+      });
+    }
+  }, {dummyData});
 
   const handleTabButtonClick = (e) => {
     const text = e.target.innerText;
     if(currentlyDisplaying === text) {
       return;
     }
     setCurrentDisplay(text);
+    text === 'SCREENSHOTS'
+    ?
+    setFeaturedMedia(dummyData.screenshots[0])
+    :
+    setFeaturedMedia(dummyData.videos[0])
+  }
+
+  const handleImageClick = (e) => {
+    console.log(e.target);
   }
 
+
   return (
     <>
       <Details text={dummyData.details.details} id={dummyData.details.id} />
       <TabButton title='VIDEOS' handleClick={handleTabButtonClick} />
       <TabButton title='SCREENSHOTS' handleClick={handleTabButtonClick}/>
-      {currentlyDisplaying === 'SCREENSHOTS' ?
-      dummyData.screenshots.map(
-        ({ link, id }) => <Img link={link} id={id} key={id} />)
-        :dummyData.videos.map(
-          ({ link, id }) => (
-        <VideoPlayer link={link} id={id} key={id} />
-      ))}
+      {
+        currentlyDisplaying === 'SCREENSHOTS'
+        ?
+        <Img link={featuredMedia.link} id={featuredMedia.id} />
+        :
+        <VideoPlayer link={featuredMedia.link} id={featuredMedia.id} />
+      }
+
+      {
+        currentlyDisplaying === 'SCREENSHOTS'
+        ?
+        <MediaCarousel mediaList={dummyData.videos} />
+        :
+        <MediaCarousel mediaList={dummyData.screenshots} />
```

Comment:
AirBnB!

Reply:
I actually havent ran it in a long time haha